project_timeline/models/project_task.py

```diff
@@ -1,13 +1,16 @@
 # Copyright 2016-2017 Tecnativa - Pedro M. Baeza
 # Copyright 2017 Tecnativa - Carlos Dauden
+# Copyright 2021 Open Source Integrators - Daniel Reis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import models
+from odoo import fields, models
 
 
 class ProjectTask(models.Model):
     _inherit = "project.task"
 
+    date_start = fields.Datetime("Start Date")
```

Comment:
To ensure avoid problems with possible function signature changes.
```suggestion
    date_start = fields.Datetime(string="Start Date")
```

Reply:
I'm sorry, I don't see the point in the change here. Using positional arguments is documented and supported, and will not change in this Odoo version.

Reply:
Not really needed, but it's always welcome to use keyword arguments as such.